java/google/registry/rde/imports/RdeImportUtils.java

```diff
@@ -66,10 +81,11 @@ public RdeImportUtils(
     this.escrowBucketName = escrowBucketName;
   }
 
-  private <T extends EppResource> void importEppResource(final T resource, final String type) {
+  public <T extends EppResource & ForeignKeyedEppResource> ImmutableSet<Object>
```

Comment:
I deliberately avoided this, because I have to store them in another collection that has objects that are represented as interfaces only. If I change it here I would need to do typecasting elsewhere.